packages/domain/common/src/valueTypes/email.valueType.ts

```diff
@@ -22,11 +22,11 @@ export const bind = ({ email }: PlainType<ValueType>): ValueType => {
 
 export const convertirEnValueType = (value: string): ValueType => {
   return bind({
-    email: value,
+    email: value?.toLowerCase(),
   });
```

Comment:
value peut pas être `undefined` 

```suggestion
    email: value.toLowerCase(),
```

Reply:
better safe than sorry... on ne sait pas quel data va venir de la DB. le type ne le permet pas mais en TS tu peux avoir un type différent au runtime

Reply:
en gros, je préfère que ca pète avec une erreur claire dans `bind` plutot qu'une sombre "cannot read toLowerCase of undefined" tu vois ? :p

Reply:
Sauf qu'on est dans le domain avec un value type, pas dans la DB.

Le type en param n'est dans undefined, donc soit faut le mettre undefined et check avant le bind dans le convertir en, soit faut faire ça avant et donc il manque une validation de surface avec un schéma. C'est pas au value type email de porter l'optionnalité, mais à l'entity dans lequel il est utilisé (Option.Type<Email.ValueType> et non email : string | undefined.